perllib/FixMyStreet/Cobrand/UK.pm

```diff
@@ -207,7 +207,11 @@ sub council_rss_alert_options {
     my ( @options, @reported_to_options );
     if ( $num_councils == 1 or $num_councils == 2 ) {
         my ($council, $ward);
-        my $body = FixMyStreet::DB->resultset('Body')->active->search({ name => { '!=' => 'TfL' } })->for_areas(keys %$all_areas)->first;
+        my $body = FixMyStreet::DB->resultset('Body')->active->search(
+            {
+                name => { '!=' => 'TfL' },
+                name => { '!=' => 'National Highways' }
```

Comment:
You can't repeat the same key in a hash, it will just overwrite - so this will have broken the existing TfL handling, I mean (add a test for that in alert as well would be good?). `name => { -not_in => ['TfL', 'National Highways'] }` should be what you want, I think.